vnext/ReactUWP/Modules/AppThemeModuleUwp.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "pch.h"
 #include "AppThemeModuleUwp.h"
+#include <iomanip>
```

Comment:
What's this for?

Reply:
The string manipulation functions for formatting the RGB values into the hex strings.